src/main/java/org/apache/commons/compress/harmony/unpack200/bytecode/forms/ClassRefForm.java

```diff
@@ -62,7 +62,7 @@ protected void setNestedEntries(final ByteCode byteCode, final OperandManager op
         // the spec except for multianewarray, which has
         // its own form.)
         final SegmentConstantPool globalPool = operandManager.globalConstantPool();
-        ClassFileEntry[] nested = null;
+        ClassFileEntry[] nested;
```

Comment:
@mawiesne 
Why is this a standalone declaration and not assigned? IOW, why not perform the assignment from the line below here? Same for the other files changed.

Reply:
> @mawiesne 
> 
> Why is this a standalone declaration and not assigned? IOW, why not perform the assignment from the line below here? Same for the other files changed.

Will check/optimize that.

Reply:
All fixed with latest push.